packages/jest/src/plugins/plugin.ts

```diff
@@ -459,3 +581,75 @@ function requireJestUtil<T>(
     paths: [dirname(resolvedJestCorePaths[jestPath])],
   }));
 }
+
+async function getJestOption<T = any>(
+  rawConfig: any,
+  absConfigFilePath: string,
+  optionName: string
+): Promise<T> {
+  if (rawConfig[optionName]) return rawConfig[optionName];
+
+  if (rawConfig.preset) {
+    const dir = dirname(absConfigFilePath);
+    const presetPath = resolve(dir, rawConfig.preset);
+    try {
+      const preset = await loadConfigFile(presetPath);
+      if (preset[optionName]) return preset[optionName];
+    } catch {
+      throw new Error(`Could not load preset file: ${presetPath}`);
```

Comment:
I think we should safely handle this and ignore this error. We can still construct the inferred project/tasks without any given option. We could warn instead.

IMHO, only fatal errors that prevent Nx from building a project should result in a graph error. We should let the tool itself handle the errors when running the tasks. It's also a matter of perspective: is Nx erroring (the graph is an Nx thing) or is the underlying tool erroring (when running the task)? Also, when the graph is in an errored state, no task, even if unrelated, can be run.

So, safely handling this would allow the task to be inferred and run. We'd be letting Jest handle the wrong preset file and error accordingly instead of us failing the graph and preventing the running of any tasks.